src/amo/components/Categories/index.js

```diff
@@ -146,12 +148,21 @@ export class CategoriesBase extends React.Component<InternalProps> {
               // $FLOW_IGNORE
               const { name, slug } = category;
 
+              const linkTo = {
+                pathname: '/search/',
+                query: convertFiltersToQueryParams({
+                  addonType: getAddonTypeFilter(addonType),
+                  category: slug,
+                  sort: `${SEARCH_SORT_RECOMMENDED},${SEARCH_SORT_POPULAR}`,
+                }),
+              };
+
               return (
                 <li className="Categories-item" key={name}>
                   <Button
                     className={`Categories-link
                       Categories--category-color-${getCategoryColor(category)}`}
-                    to={`/${visibleAddonType(addonType)}/${slug}/`}
```

Comment:
Re: https://github.com/mozilla/addons-frontend/issues/8079#issuecomment-496551743 , you could do the redirects and removal of [`pages/CategoriesPage`](https://github.com/mozilla/addons-frontend/blob/1d67671dd5004fc6e894ebd55ca97766233d8c43/src/amo/pages/CategoriesPage/index.js) in a follow-up.

Reply:
I opened https://github.com/mozilla/addons-frontend/issues/8090 for this. Is this what you mean, or is there something else you think needs to be done in a follow-up. Note there are a lot of open issues right now around all of this stuff, but I want to make sure I don't land anything that's missing some required changes.